src/api/routes/test.ts

```diff
@@ -0,0 +1,92 @@
+import AsyncRouter from '../AsyncRouter';
+import { Response } from 'express';
+import {
+  createOrUpdateTest,
+  getUser,
+  getTests,
+} from '../../application/service';
+import { ApiError } from '../ApiError';
+import { User } from '../../domain/model/user/User';
+//import {Test as ApiTest} from '../interface/index';
+import { Test } from '../../domain/model/test/Test';
+
+import logger from '../../logger';
+import { isAuthenticated } from '../middleware/isAuthenticated';
+import { AuthenticatedRequest } from '../AuthenticatedRequest';
+
+export default () => {
+  const route = new AsyncRouter();
+
+  route.get(
+    '/users/:id/tests',
+    isAuthenticated,
+    async (req: AuthenticatedRequest, res: Response) => {
+      const { id } = req.params;
+      const user = await getUser.byId(id);
+
+      if (!user) {
+        throw new ApiError(404, 'user.not-found');
+      }
+
+      const tests = await getTests.byUserId(id);
+
+      return res
+        .json(tests.map((test: any) => mapTestToApiTest(test)))
+        .status(200);
+    }
+  );
+
+  route.get(
+    '/tests/:id',
```

Comment:
Do we have a usecase for this one?

Reply:
I added this as we could have a flow in the future where clinicians/admins have a screen where they punch in testID or we have QRCodes/barcodes to scan which gives testId to look up.